cache_images/ubuntu_packaging.sh

```diff
@@ -20,12 +20,24 @@ lilto ooe.sh $SUDO apt-get -qq -y update
 bigto ooe.sh $SUDO apt-get -qq -y upgrade
 
 echo "Configuring additional package repositories"
-lilto ooe.sh $SUDO add-apt-repository --yes ppa:criu/ppa
+
+
+if [[ "$OS_RELEASE_VER" -le 2004 ]]; then
+
+    # Needed for criu package
+    lilto ooe.sh $SUDO add-apt-repository --yes ppa:criu/ppa
+fi
+
+# The versions of some key dependencies are not updated quickly
+# enough to support upstream containers-org development.
 VERSION_ID=$(source /etc/os-release; echo $VERSION_ID)
-echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_$VERSION_ID/ /" \
-    | ooe.sh $SUDO tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
-ooe.sh curl -L -o /tmp/Release.key "https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_${VERSION_ID}/Release.key"
-ooe.sh $SUDO apt-key add - < /tmp/Release.key
+REPO_URL="https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/testing:/ci/xUbuntu_$VERSION_ID/"
+GPG_URL="https://build.opensuse.org/projects/devel:kubic:libcontainers:testing:ci/public_key"
```

Comment:
Are you sure it's `public_key`? If you look at https://software.opensuse.org//download.html?project=devel%3Akubic%3Alibcontainers%3Atesting%3Aci&package=runc , it still says `Release.key` and not `public_key`.

Reply:
fixed